index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <nav>
+                <a href=""><img src="./assets/images/Logo Placeholder.png" alt=""></a>
+                <a href="http://"><img src="./assets/images/Enabled.png" alt="my logo"></a>
+            </nav>
+        </header>
+
+        <section>
+            <h1>
+                I'm Abdy.<br /> Glad to see <br /> you!
+            </h1>
+            <div class="content">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me.</p>
+            </div>
+            <h2 class="connect-link">LET'S CONNECT</h2>
+            <div class="social-media-icons">
+                <ul class="social-media-links">
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/twiter.png" alt="twiter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/Linkedin icon.png" alt="linkedin">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/medium.png" alt="medium">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/git.png" alt="git">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="http://">
+                            <img src="./assets/images/vector.png" alt="angel">
+                        </a>
```

Comment:
- [ ] As previously mentioned, the usage of href='http://' creates an issue when clicking on the social media link. To resolve this, you can either utilize your specific social media link or use href='#'. This will ensure that the link functions correctly. 😍🎉